### DISCUSSION TITLE
essay/2024/02/14/test-code

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#173] [Comments]
# essay/2024/02/14/test-code

테스트 코드를 짜는 게 이렇게 즐거운 건지 몰랐습니다. 테스트를 설계하고 이를 구현해서 내가 원하는 대로 잘 동작하는 걸 확인하면 얼마나 큰 행복감이 몰려오는지.

https://jeho.page/essay/2024/02/14/test-code.html

**Comment by voidFunctionReturn0** [comment]
이미 보셨을테지만, flutter test 예제를 공유드려봅니다

https://docs.flutter.dev/testing/overview
